docs/proposals/20210310-opt-in-autoscaling-from-zero.md

```diff
@@ -204,8 +245,16 @@ status:
     memory: 500mb
     cpu: "1"
     nvidia.com/gpu: "1"
+  nodeInfo:
+    architecture: arm64
+    operatingSystem: linux
 ```
 
+The information stored in the `status.nodeInfo` field is rendered as labels for the node object that is created within
+the node group by the cluster autoscaler and fed into the cluster autoscaler's scheduler simulator `framework.NodeInfo` struct. 
+In particular, the `architecture` and `operatingSystem` fields are used to determine the simulated node's labels 
+`kubernetes.io/arch` and `kubernetes.io/os`. This logic will be implemented in the cluster autoscaler's ClusterAPI cloud provider code.
```

Comment:
What about simplifying into 
```suggestion
The information stored in the `status.nodeInfo` field will be used by the cluster autoscaler's scheduler simulator to determine the simulated node's labels `kubernetes.io/arch` and `kubernetes.io/os`. This logic will be implemented in the cluster autoscaler's ClusterAPI cloud provider code.
```

(I don't think we need to surface autoscaler implementation details in this proposal)